### PR TITLE
Created Hindi Translation

### PR DESCRIPTION
**Pull request by RadioactiveRadio**

### CODE REVIEW

File: po/hi.po [hunk 1]
```diff
@@ -0,0 +1,611 @@
...
+#: paleta/ui/window.ui:37
+msgid "Edit Palettes"
+msgstr "पैलेट्स एडिट करें"
+
+#: paleta/ui/window.ui:58
+msgid "Drop"
+msgstr "ड्रॉप करें"
+
+#: paleta/ui/window.ui:69
+#, fuzzy
```
Comment (gregorni) [inline po/hi.po:149]: I think this line needs to be removed.

Comment (nate-xyz) [reply po/hi.po:149]: I'm not sure, I think it autogenerates in the potfile. The other translation po files have it

Comment (gregorni) [reply po/hi.po:149]: Oh, in that case nevermind.

Comment (RadioactiveRadio) [reply po/hi.po:149]: Can confirm, i read the whole thing. It wasn't there before.

**Review by nate-xyz — CHANGES_REQUESTED** [review]

**Pull request merged**